quantpy/sympy/executor/classical_simulation_executor.py

```diff
@@ -35,7 +35,7 @@ def execute(self, circuit, **options):
         qobj = circuits_to_qobj(circuit, QasmSimulatorPy())
         json = qobj_to_dict(qobj)["experiments"][0]
         self.simulate(json)
-        return str(self.simulator)
+        return self.simulator.to_coefficients()
```

Comment:
is this change acceptable? @kyamaz 
previously this was a string like `0.5|0> + 0.5|1>`, but changed to `{'0': 0.5, '1': 0.5}`

I want to use this result in qexperiment.

Reply:
I think it is pretty good to change output format of execute function in ClassicalSimulationExecutor. @ssmi1975

Reply:
thanks! I will keep this then.